pkg/cmd/extension/command.go

```diff
@@ -293,19 +293,35 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				Use:   "install <repository>",
 				Short: "Install a gh extension from a repository",
 				Long: heredoc.Docf(`
-					Install a GitHub repository locally as a GitHub CLI extension.
+					Install a GitHub CLI extension from a GitHub or local repository.
```

Comment:
I don't think there's anything that requires the local directory to be a repository, so this doesn't seem quite right.

That said I understand that the short and use indicate that it's a repository, so let's just leave it alone.

Reply:
I had this thought too and here is how I rationalized it to myself:

I saw us using `repository` to describe the directory as a suggestion that we want extension authors to consider their extensions as repositories, even if we don't strictly require it. We especially want them to consider them as _GitHub_ repositories, except in this case it is cloned locally.

Maybe there's holes on my logic 🤪 , but that was my thoughts and why I didn't change it.

I'll mark this thread as resolved, but feel free to re-open it if you think we have something to do here 🙏